### DISCUSSION TITLE
Funneling comments/flags from CalFlora to specimens in CCH2

### DISCUSSION THREAD

**Discussion by Annie-Ayers** [Symbiota/community-board#591] [Wish List]
It would be very helpful to have a more streamlined method of viewing the comments of herbarium specimens from CalFlora directly in CCH2. Its very difficult to address the flagged specimens in CalFlora and document which comments have been addressed. Often these comments relate to quality control that is important to change.

**Comment by umn-annakfowler** [comment]
We have a similar thought around comments - is there potential for an avenue for notifying collection managers about new comments? The Bell Museum Herbarium (MIN) collections have been harvested by many Symbiota portals as a snapshot, and we don't know there are comments on these portals unless someone emails us directly. 

Are there other collections in the same boat where they lack the time/bandwidth to check every portal for comments, but would love to address the comments and update records where needed? 

We are thinking mostly of a simple notification that comments exist over including comments in the notification.

Alternatively, perhaps another option would be to disable comments as part of the general snapshot collection settings? That might prevent well-meaning contributors from scattering comments about a record on the various aggregators vs. directly contacting the collection manager.

Another aspect of this is that the current comment system doesn't record (on the front-end) who 'reviews' each comment or the date on which it was reviewed; and doesn't allow you to record any actions taken along with the review (e.g. "Updated Record"). This has led to a few situations where comments were 'reviewed' for posting but changes were never made to the records.
